docs/installation.md

```diff
@@ -58,11 +58,22 @@ The following instructions are for setting up a version of Amundsen using Docker
     $ docker-compose -f docker-amundsen.yml down
     ```
 
-## Verify setup or troubleshooting
+## Verify setup
```

Comment:
nit: We should indent these with `###` since its only applicable to the Docker setup